src/argilla/client/feedback/training/frameworks/trl.py

```diff
@@ -204,7 +286,29 @@ def preprocess_function(examples) -> Dict[str, List]:
                 tokenizer=self._transformers_tokenizer,
                 **self.trainer_kwargs,
             )
+        elif isinstance(self._task, TrainingTaskForPPO):
+            from datasets import concatenate_datasets
+
+            dataset = concatenate_datasets([x for x in [self._train_dataset, self._eval_dataset] if x is not None])
+
+            def tokenize(sample):
+                sample["input_ids"] = self._transformers_tokenizer.encode(sample["text"])
+                sample["query"] = self._transformers_tokenizer.decode(sample["input_ids"])
```

Comment:
Is there a difference between this and just `sample["text"]`?

Reply:
Nope, this was some leftover form some code I copied from TRL and forgot to remove it.